setup.py

```diff
@@ -36,6 +38,7 @@ def find_version(*file_paths):
     url = "https://github.com/bram85/topydo",
     install_requires = [
         'six >= 1.9.0',
+        'arrow >= 0.6.0',
```

Comment:
`arrow` 0.7 hasn't hit PyPi yet, so I'll test it once it's released there. In any case, do we want the requirements listed here to the current version, or the earliest known working version?